src/main/scala/devices/uart/UART.scala

```diff
@@ -18,16 +18,19 @@ case class UARTParams(
   oversample: Int = 4,
   nSamples: Int = 3,
   nTxEntries: Int = 8,
-  nRxEntries: Int = 8)
+  nRxEntries: Int = 8,
+  wire4: Boolean = false)
 {
```

Comment:
Could you also push your real-time configurable 8/9 bit data frame changes here?